### PR TITLE
(MAINT) Repair Analytics Reporting Regression

### PR DESCRIPTION
**Pull request by jfryman**
This PR fixes an issue introduced in https://github.com/puppetlabs/lumogon/commit/d128444ce0da6c5540a2cf78b6a14b15bd35439b where the correct Viper bind is not used for the `disable-analytics` endpoint. As a result, this has been ignored and all reports have been sending analytics data upstream regardless of flag being set.

### CODE REVIEW

**Comment by jfryman** [comment]
Correct codepath being followed here. 

```
[lumogon] 2017/05/24 18:28:57.082966 [Analytics] Gathering analytics for screen-view, report
[lumogon] 2017/05/24 18:28:57.082993 [Docker Adapter] Creating container runtime client: Docker
[lumogon] 2017/05/24 18:28:57.135309 [Scheduler] Creating scheduler
[lumogon] 2017/05/24 18:28:57.135332 [Docker Adapter] Creating container runtime client: Docker
[lumogon] 2017/05/24 18:28:57.135342 [Scheduler] Running
[lumogon] 2017/05/24 18:28:57.135346 [Scheduler] Creating context with timeout [60]
[lumogon] 2017/05/24 18:28:57.135416 [Analytics] Skipping submission of Google Analytics event
```

We need to test this in E2E once we get those up and going.

File: analytics/ga.go [hunk 1]
```diff
@@ -60,6 +60,7 @@ func NewUserSession() *UserSession {
 
 // ScreenView is the public function to post a ScreenView analytics message to GA
 func ScreenView(screen string) {
+	logging.Stderr("[Analytics] Gathering analytics for screen-view, %s", screen)
```
Comment (johnmccabe) [inline analytics/ga.go]: perhaps rephrase screen-view, user probably won't know about Google Analytics? This is just initialising the GA session, so perhaps something like that

Comment (jfryman) [inline analytics/ga.go]: https://github.com/puppetlabs/lumogon/pull/37/files#diff-a152202c6575ec6127abb26b8064f836

File: analytics/ga.go [hunk 2]
```diff
@@ -60,7 +60,7 @@ func NewUserSession() *UserSession {
 
 // ScreenView is the public function to post a ScreenView analytics message to GA
 func ScreenView(screen string) {
-	logging.Stderr("[Analytics] Gathering analytics for screen-view, %s", screen)
+	logging.Stderr("[Analytics] Initializing Google Analytics for screen: %s", screen)
```
Comment (johnmccabe) [inline analytics/ga.go]: dump the `for screen`

/disengage pedant mode

Comment (jfryman) [inline analytics/ga.go]: I :heart: you outright rejected the pull based on this. :)

**Comment by johnmccabe** [comment]
@jfryman I'll 🚢 when green

**Comment by tylerpace** [comment]
👍

**Pull request merged**